client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -1613,6 +1641,21 @@ export default {
       }
     },
 
+    getStorageRecordListFilterPreset () {
+      try {
+        // Get record list filters from localStorage
+        const currentFilters = getItem(`record-list-preset-${this.uniqueID}`)
+
+        // Set the custom preset filters
+        this.customPresetFilters = [currentFilters]
```

Comment:
isn't currentFilters and array? That would mean you would have to use [...currentFilters], i might be looking at this wrong tho

Reply:
This format follows the same way the preset filter are saved on the options object, but in this case all the custom filter are the same